src/Error/ExceptionRendererInterface.php

```diff
@@ -21,7 +21,7 @@
 /**
  * Interface ExceptionRendererInterface
  *
- * @method \Psr\Http\Message\ResponseInterface|string render() Render the exception to a string or Http Response.
+ * @method \Psr\Http\Message\ResponseInterface render() Render the exception to a string or Http Response.
```

Comment:
@markstory I am bit confused here. The annotation says `render()` can return a string but that's not possible given that it already has return type specified as `ResponseInterface`,

Reply:
Understood, so I will keep the annotation unchanged. 

But that makes static analyzers unhappy since ErrorHandlerMiddleware::handleException() assumes that the return value will always be a response instance:
https://github.com/cakephp/cakephp/blob/b52641a6a210fe1a80d13dd93ad9d1085cdcff52/src/Error/Middleware/ErrorHandlerMiddleware.php#L148

In practice it will be a response instance since we would only use the HtmlRenderer for web practice. So should I just add an inline `@var` annotation to denote is as response instance?

Reply:
> So should I just add an inline @var annotation to denote is as response instance?

I can do that and update this pull request.

Reply:
I have taken care of it :slightly_smiling_face: